subsys/fs/littlefs_fs.c

```diff
@@ -595,6 +595,7 @@ static int littlefs_mount(struct fs_mount_t *mountp)
 	if (block_size == 0) {
 		block_size = get_block_size(fs->area);
 	}
+	__ASSERT_NO_MSG(block_size > 0);
```

Comment:
This won't make you skip the coverity report of divide-by-zero, imho.
If this should never happen, could we return after the ASSERT?

Reply:
Well, I guess.  There's lots of places in Zephyr where if you make it past an ASSERT you're going to blow up.  And really I don't think you can have a flash area with no regions with a positive block size.   But since we're trying to placate Coverity I think the rework should do it.